module-1/lab-api-scavenger-game/your-code/Challenge2.ipynb

```diff
@@ -0,0 +1,2137 @@
+{
```

Comment:
In this challenge you have done the JSON normalization!! Why haven't you finished? Just another line more and you had it!! Would you like us to take a look on it?